week1/Mauricio_Yeverino/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>You've been Rao'd</title>
+  <link rel="stylesheet" href="style/index.css">
+</head>
+<body>
+  <h1>
+    You've been Roa'd
+  </h1>
+  <p>
+      The goat, don't click on Rao
+  </p>
+  <div class="main-section">
+    <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ&pp=ygUjcmljayBhc3RsZXkgbmV2ZXIgZ29ubmEgZ2l2ZSB5b3UgdXA%3D">
+      <img src="img/rao.jpg">
+    </a>
+    <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ&pp=ygUjcmljayBhc3RsZXkgbmV2ZXIgZ29ubmEgZ2l2ZSB5b3UgdXA%3D">
+      <img src="img/LebronJames.jpg">
```

Comment:
```suggestion
      <img src="/img/rao.jpg">
    </a>
    <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ&pp=ygUjcmljayBhc3RsZXkgbmV2ZXIgZ29ubmEgZ2l2ZSB5b3UgdXA%3D">
      <img src="/img/LebronJames.jpg">
```
This might fix your images?